File: week04-2.py
#week04-2.py 是今天LeetCode挑戰題
#LeetCode 2529. Maximum Count of Positive Integer and Negative Integer
#找出(有幾個正數)(有幾個負數)大的那個數量回傳
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        pos=0
        neg=0 #迴圈前面準備好都放0
        for i in range (len(nums)):#迴圈裡更新它
            if nums[i]>0:pos+=1
            if nums[i]<0:neg+=1
        #迴圈後面拿來用
        return max(pos,neg)

File: week04-2.c
//week04-2.c さぱLEETCODE珼驹肈
//LEETCODE 2529.Maximon COunt Positive
int maximumCount(int* nums, int numsSize) {

    int pos =0,neg = 0;//癹伴玡常琌0

    for (int i=0; i<numsSize; i++){
        if (nums[i]>0)pos++;//タ计
        if (nums[i]<0)neg++;//璽计
    }//癹伴い丁穝 э

//癹伴рウㄓノ
    if (pos>neg)return pos;//タ计ゑ
    else return neg;//ぃ礛碞璽计
}
